test/integration/features/basic.feature

```diff
@@ -29,6 +29,10 @@ Feature: Basic
     Given Minishift has state "Does Not Exist"
      When executing "minishift start" succeeds
      Then Minishift should have state "Running"
+      And image export completes with 3 images
+      And container image "openshift/origin-haproxy-router:v[0-9]+\.[0-9]+\.[0-9]+" is cached
+      And container image "openshift/origin-docker-registry:v[0-9]+\.[0-9]+\.[0-9]+" is cached
+      And container image "openshift/origin:v[0-9]+\.[0-9]+\.[0-9]+" is cached
```

Comment:
@amitkrout , @agajdosi  FYI from downstream we need to change it for sure because images have different name and tags

Reply:
@praveenkumar ack. Once this pr is merged, will create a jira and change it accordingly in downstream